### PR TITLE
bluetooth-audio: initial add-on

### PR DESCRIPTION
**Pull request by lrusak**
allows switching to and from a bluetooth audio device when connecting and disconnecting

Works well, switches from the current audio device to the pulse device, back to the original audio device.

Maybe in the future I'll add the ability to add setting to set an audio device, but it may not be needed.

I've added the ability to support multiple devices, however, in my testing I cannot connect more than one bluetooth audio device at a time :man_dancing: 

needs an icon courtesy of @chewitt 

EDIT:

one more thing. I explicitly look for the bluetooth device class to check for an audio type device. I'll have to see how that goes and we might have to narrow it down more.

I found this awesome page to test the bitmask needed, http://bluetooth-pentest.narod.ru/software/bluetooth_class_of_device-service_generator.html

SECOND EDIT:

Tested with a bluetooth remote and it doesn't switch the audio device, so it looks like the bitmask is working properly :+1:

### CODE REVIEW

**Comment by lrusak** [comment]
Yay! We have an icon now!

**Comment by lrusak** [comment]
test add-on for anyone wanting to try. http://lrusak.libreelec.tv/public/service.bluetooth-audio-9.0.100.zip

**Comment by lrusak** [comment]
I'm happy with this now :+1:

File: packages/addons/service/bluetooth-audio/source/resources/language/English/strings.po [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Kodi Media Center language file
+# Addon Name: docker
+# Addon id: service.system.docker
```
Comment (chewitt) [inline packages/addons/service/bluetooth-audio/source/resources/language/English/strings.po]: ??

Comment (lrusak) [inline packages/addons/service/bluetooth-audio/source/resources/language/English/strings.po]: whoops

**Pull request merged**